docsrv/fetcher_test.go

```diff
@@ -0,0 +1,28 @@
+package docsrv
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testOwner   = "erizocosmico"
```

Comment:
This needs real usage of the GitHub API, so yeah, it's a real repo. What do they have?